packages/cursorless-everywhere-talon-core/src/types/Global.d.ts

```diff
@@ -0,0 +1,8 @@
+declare function print(...objs: any): void;
```

Comment:
what happens when you run in node during testing?

Reply:
We don't call print. Print only exits so our polyfill is allowed to shunt console log to it.

Reply:
But the poly fill is defined in a different package right?